### ISSUE TITLE
Cmoa Manga with pages in different resolutions causing problems

### ISSUE THREAD

**Issue by KremeDelaKrem** [xuzhengyi1995/Manga_downloader#117]
I've noticed a new problem with downloading some manga that some pages end up being stitched together incorrectly.  For example, half the pages will be in 1131x1616 and the other half will be in 1440x2048 and all the 1131x1616 would be stitched incorrectly.  I managed to fix this by changing the mmset in cmoa_jp_actions and downloading the manga twice but it there a way to do it so I only have to run the code once.

**Comment by xuzhengyi1995** [comment]
Could I have a example manga URL to figure out the problem? The free one is better because I could not purchase on coma. Thanks.

**Comment by KremeDelaKrem** [comment]
Here's the free sample of the manga I was talking about.
https://www.cmoa.jp/bib/speedreader/?cid=0000234506_jp_0002&u0=1&u1=0&rurl=https%3A%2F%2Fwww.cmoa.jp%2Ftitle%2F234506%2F

But I have found another problem.
The images in the free sample are different to the ones in my paid copy.
For example, the cover.
![e1f41cd6964778fa857c59eff1e1a038](https://user-images.githubusercontent.com/119675220/205433431-9fc42ee7-efef-406a-a19a-1592e255d1bd.png)
![21ad49a30209a242acd160b00de9e1ed](https://user-images.githubusercontent.com/119675220/205433438-d86b2aea-c86e-49f5-8102-b2a517482575.png)
The top one is from the free sample and the bottom one is from my purchased copy.

And here's page 6.
![a75a4fdae0369625781310e1c11f7c96](https://user-images.githubusercontent.com/119675220/205433624-fe5bb98d-01aa-4cdb-a4e0-f5e52fce34d3.png)
![f5a4fbd11776de31a5e7fb3da79f7df8](https://user-images.githubusercontent.com/119675220/205433625-2bfed805-19fd-4e7d-8044-55f9f9e31f4a.png)
Top free, bottom paid.

You can see that my version has different intrinsic sizes.

This causes me problems when I use cmoa_jp_actions on default mmset.
Here's the cover I get.
![image](https://user-images.githubusercontent.com/119675220/205433817-afb6d0bd-4a01-4dd8-8056-ac3275e3c4df.png)
I circled where the image gets messed up.

If you need a free sample where the images are in different resolutions I can try to find one.

Thank you for your help.

**Comment by flapjack3** [comment]
> I've noticed a new problem with downloading some manga that some pages end up being stitched together incorrectly. For example, half the pages will be in 1131x1616 and the other half will be in 1440x2048 and all the 1131x1616 would be stitched incorrectly. I managed to fix this by changing the mmset in cmoa_jp_actions and downloading the manga twice but it there a way to do it so I only have to run the code once.

Yeah I've come across this issue many times too. Didn't even realize it was 1131x1616 pages doing this until I saw this comment and checked my files. Would you be able to explain how you're changing the mmset in cmoa_jp_actions to work around this for now? (I'm a newbie at this, sorry..)
